rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
Actually technically the dependency is `net-libs/nodejs[npm]`, that is, the npm USE flag has to be enabled on it. But I don't know if your base.yaml gentoo key value supports any of that